lib/src/server/sse_handler.dart

```diff
@@ -13,7 +13,7 @@ import 'package:shelf/shelf.dart' as shelf;
 import 'package:stream_channel/stream_channel.dart';
 
 // RFC 2616 requires carriage return delimiters.
-String _sseHeaders(String origin) => 'HTTP/1.1 200 OK\r\n'
+String _sseHeaders(String? origin) => 'HTTP/1.1 200 OK\r\n'
```

Comment:
If origin is null, would "Access-Control-Allow-Origin: null\r\n" would cause problems?